websites/docs-smoke-test/src/content/views/beta.mdx

```diff
@@ -0,0 +1,4 @@
+---
+title: Beta page
+beta: true
```

Comment:
The homepage does not use the "beta" flag. For VRT purposes we have a new page now.